lib/collectionspace/converter/core/organization.rb

```diff
@@ -11,7 +11,7 @@ def convert
                 "xmlns:xsi" => "http://www.w3.org/2001/XMLSchema-instance"
             ) do
               xml.parent.namespace = nil
-              CoreOrganization.map(xml, attributes, config)
+              CoreOrganization.map_common(xml, attributes, config)
```

Comment:
@Paulmulonzia - Since they can be reused in other modules, Mark and I decided it is best to be explicit, specific, and consistent when naming each mapping method. Thus, in each core mapper (person, org, collectionobject, etc), the main mapping method should be called `map_common`

This will probably be inconsistent for a while, but can be updated as we add modules that call these methods.

Reply:
Got it